xiilib/databases.py

```diff
@@ -73,20 +75,9 @@ def fetch_relation_data(
         Returns: required relation data.
         """
         data = {}
-        for relation in self._charm.model.relations[self.relation_name]:
-            endpoints = []
-            if relation.app is None:
-                continue
-            for unit in relation.units:
-                if not (hostname := relation.data[unit].get("hostname")):
-                    continue
-                if not (port := relation.data[unit].get("port")):
-                    continue
-                endpoints.append(f"{hostname}:{port}")
-            if endpoints:
-                data[relation.id] = (
-                    {"endpoints": ",".join(endpoints)} if "endpoints" in fields else {}
-                )
```

Comment:
Is this no longer required for all the databases we support?

Reply:
This is only used for redis so it can be removed now.

Reply:
The function is called `fetch_relation_data` which seems to indicate that it fetches relation data for all the databases?